### PR TITLE
Require PHP 7.1

### PR DESCRIPTION
**Pull request by lcobucci**

### CODE REVIEW

File: src/Signer/Rsa.php [hunk 1]
```diff
@@ -52,7 +52,7 @@ public function verify(string $expected, string $payload, Key $key): bool
      *
      * @expectedException InvalidArgumentException
```
Comment (Ocramius) [inline src/Signer/Rsa.php]: This is probably a mistake/leftover

File: .travis.yml [hunk 2]
```diff
@@ -2,12 +2,11 @@ language: php
 sudo: false
 
 php:
-  - 7.0
   - 7.1
   - nightly
 
 before_script:
-  - composer install --prefer-dist -o
+  - composer install -o
```
Comment (Ocramius) [inline .travis.yml]: Don't use `-o` for tests, as it will cause the test suite to succeed even if there is a misnamed class

File: composer.json [hunk 3]
```diff
@@ -47,6 +47,9 @@
             ]
         }
     },
+    "config": {
+        "preferred-install": "dist"
```
Comment (Ocramius) [inline composer.json]: Oh, that's new :O

Comment (lcobucci) [inline composer.json]: AHAHAHA I don't think it's THAAAAT new 😜

File: src/Signer/Rsa.php (hunk 1 above)
Comment (lcobucci) [inline src/Signer/Rsa.php]: Definitely

**Comment by manchuck** [comment]
Just out of curiosity, why php 7.1 and not 7.0 ?   I do not see any php 7.1 features being used.  I only ask because I am currently running 7.0 using 4.0.0.  This is now causing some issues with our build process

**Comment by manchuck** [comment]
NVM found the nullable type for getType on the builder

**Pull request merged**